src/ORM/DataQuery.php

```diff
@@ -1481,7 +1481,8 @@ public function getCacheKey(): string
      */
     public function setQueryParam($key, $value)
     {
-        $this->queryParams[$key] = $value;
+        $k = $key === null ? '' : $key;
+        $this->queryParams[$k] = $value;
```

Comment:
```suggestion
        $this->queryParams[$key ?? ''] = $value;
```
Either this or `$k = $key ?? '';` but I think my first suggestion is cleaner.

Using the ternary operator here is confusing - it took me a few seconds to realise it's just doing that `??` does and even while writing this I find myself re-reading the code in case I missed some weird thing that's happening here.

Reply:
Updated